src/snowflake/snowpark/modin/plugin/docstrings/series.py

```diff
@@ -3277,14 +3277,6 @@ def take():
         3    2
         dtype: int64
 
-
-        For `Series` axis parameter is unused and defaults to 0.
-
-        >>> ser.take([0, 3], axis=1)
```

Comment:
why this is removed now?

Reply:
I see, do you know if it is a new behavior introduced in 2.2.* compare with 1.5.3

Reply:
Yes, seems like it works in pandas 2.0.3 and fails in pandas 2.1.4.

Reply:
Can you add a entry in the change log to indicate this difference?

Reply:
done